tools/wltests/test_series

```diff
@@ -380,9 +380,13 @@ energy_measurement:
     # If collecting on multiple channels, or another channel than iio:device0,
     # set them here:
     iio_devices: [$IIO_DEVICES]
+
+# Automatically generated ADB device configuration based on test_series options
```

Comment:
nit-pick: usually a `device_config` section comes at the beginning of an agenda file.
Can you move this at the beginning of the `config_local.yaml`?

Reply:
Done